conf/messages

```diff
@@ -805,4 +805,4 @@ accessibility.statement.heading.eight = How we tested this service
 full.stop= .
 
 ############################### Footer ###############################
-footer.links.accessibility.text = Accessibility
+footer.links.accessibility.text = Accessibility statement
```

Comment:
This change should not be made yet.  To start with we want our footer to match up with what we believe ATOS are implementing (the design we will have given them).

Reply:
done